### ISSUE TITLE
Cannot set location of calibre database

### ISSUE THREAD

**Issue by aeroaks** [janeczku/calibre-web#1828]
**Describe the bug/problem**
I have an existing location containing one calibre library with metadata.db. I have tried the AUR installer and docker setup from linuxserver. But for both I get `DB Location is not Valid, Please Enter Correct Path` when I enter the location of the folder containing the database.

**To Reproduce**
1. Go to 'http://localhost:8083'
2. Redirects to 'http://localhost:8083/basicconfig'
3. Enter location to calibre library, without 'metadata.db'

**Logfile**
[2021-01-25 21:32:18,386]  INFO {cps:101} Starting Calibre Web...
[2021-01-25 21:32:18,478]  INFO {cps.server:168} Starting Tornado server on :8083
[2021-01-25 21:33:11,562]  WARN {cps.config_sql:335} invalidating configuration
[2021-01-25 21:33:33,567]  WARN {cps.config_sql:335} invalidating configuration
[2021-01-25 21:33:36,660]  WARN {cps.config_sql:335} invalidating configuration
[2021-01-25 21:34:22,677]  WARN {cps.config_sql:335} invalidating configuration
[2021-01-25 21:34:27,935]  WARN {cps.config_sql:335} invalidating configuration
[2021-01-25 21:35:07,721]  WARN {cps.config_sql:335} invalidating configuration

**Expected behavior**
Expected that the location is taken without error.

**Screenshots**
![image](https://user-images.githubusercontent.com/3954144/105764889-e002b780-5f57-11eb-93f9-1f0e27288ce0.png)

**Environment (please complete the following information):**
 - OS: Manjaro 20
 - Python version: python 3.9.1
 - Calibre-Web version: 0.6.10
 - Docker container: LinuxServer
 - Systemctl: AUR package
 - Special Hardware: x86
 - Browser: Firefox 84.0.2

**Additional context**
have tried both linuxserver docker and aur package. both give the same error.

**Comment by OzzieIsaacs** [comment]
I can only tell you this message means that calibre-web isn't finding a metadata.db file in the folder you entered. Many users using Docker containers having problems entering the path, please ask for help at the docker container repository, as I have no clue about this docker folder mapping stuff

**Comment by aeroaks** [comment]
Hi, i tried docker initially and then tried installation using the package manager , without docker. Both had the same error.

**Comment by freshyill** [comment]
@aeroaks I ran into the same thing, and it was a permissions issue with the group. Check your user id and groups. 

I had put `100` for the `users` GUID, but the the `users` group does not have access to the directory I want to use. I changed the GUID to `101` for administrators, and this fixed it (obviously yours may vary)

Also, it's unclear if "location" means the _location_ of the db file or the _path to the db file_, but it wants the path. I used just `/books`. Hope this helps!
